src/modules/authorizer/role/AUT_Roles_v1.sol

```diff
@@ -50,11 +50,7 @@ contract AUT_Roles_v1 is
 
     //--------------------------------------------------------------------------
     // Storage
-
-    // Stored for easy public reference. Other Modules can assume the following roles to exist
-    bytes32 public constant ORCHESTRATOR_OWNER_ROLE = "0x01";
     bytes32 public constant ORCHESTRATOR_MANAGER_ROLE = "0x02";
```

Comment:
Also wouldnt the ORCHESTRATOR_MANAGER_ROLE move up to 0x01?

Reply:
No, because I'm going to remove it in the next PR :)